### PR TITLE
feat: Add stetho to project

### LINKED ISSUE #1138: Add stetho to app
**Describe the chore**
Add stetho to the app for remote introspection

**Would you like to work on the issue?**

- [ ] Yes
- [x] No
- Other:

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1138 

Changes: 

Added stetho to project.

(Also fixed indentation of some lines)

### CODE REVIEW

**Comment by ShridharGoel** [comment]
@iamareebjamal Should I create a separate PR to fix indentation or it's okay here?

File: app/src/main/java/org/fossasia/openevent/general/OpenEventGeneral.kt [hunk 1]
```diff
@@ -28,5 +29,13 @@ class OpenEventGeneral : MultiDexApplication() {
         ))
         Timber.plant(Timber.DebugTree())
         AndroidThreeTen.init(applicationContext)
+
+        if (BuildConfig.DEBUG) {
+            Stetho.initialize(
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/OpenEventGeneral.kt]: Just use initialize with defaults

File: app/src/main/java/org/fossasia/openevent/general/di/Modules.kt [hunk 2]
```diff
@@ -235,3 +237,7 @@ val databaseModule = module {
         database.orderDao()
     }
 }
+
+fun stethoInterceptor(): StethoInterceptor {
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/di/Modules.kt]: No need of another function for this

File: app/src/main/java/org/fossasia/openevent/general/OpenEventGeneral.kt (hunk 1 above)
Comment (ShridharGoel) [reply app/src/main/java/org/fossasia/openevent/general/OpenEventGeneral.kt]: Done

File: app/src/main/java/org/fossasia/openevent/general/di/Modules.kt (hunk 2 above)
Comment (ShridharGoel) [reply app/src/main/java/org/fossasia/openevent/general/di/Modules.kt]: Removed

**Pull request merged**